src/frontend/src/common/constants/inputDelimiters.ts

```diff
@@ -1 +1,2 @@
-export const INPUT_DELIMITERS = /[\n\t,]/g;
+export const INPUT_DELIMITERS = /[\n\t, ]/g;
```

Comment:
@vincent-czi You might want to check other sites where this is used to ensure the functionality of the inputs aren't affected.

Reply:
Good catch! Yeah, I realized over the weekend that might be a problem, sure enough it is. I need to make a PR and split out two versions of the delimiters.